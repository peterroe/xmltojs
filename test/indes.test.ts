import { describe, expect, it } from 'vitest'
import xmltojs from '../src/index'

describe('test', () => {
  it('should workd', () => {
    const str = `<MyRoot><test>Success</test><test2><item>val1</item><item>val2</item></test2></MyRoot>`
    expect(xmltojs(str)).toMatchSnapshot(``)
  })
})
