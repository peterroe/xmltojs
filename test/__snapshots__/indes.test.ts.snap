// Vitest Snapshot v1

exports[`test > should workd 1`] = `
{
  "HTML": [
    {
      "HEAD": "",
    },
    {
      "BODY": [
        {
          "MYROOT": [
            {
              "TEST": "Success",
            },
            {
              "TEST2": [
                {
                  "ITEM": "val1",
                },
                {
                  "ITEM": "val2",
                },
              ],
            },
          ],
        },
      ],
    },
  ],
}
`;
